WORKSPACE

```diff
@@ -161,3 +161,10 @@ http_archive(
     strip_prefix = "hadoop-3.3.1",
     urls = ["https://dlcdn.apache.org/hadoop/common/hadoop-3.3.1/hadoop-3.3.1.tar.gz"],
 )
+
+http_archive(
+    name = "json",
+    build_file = "//config:json.BUILD",
+    strip_prefix = "json-3.11.2",
```

Comment:
why there is no sha256 field?

Reply:
There is no hash provided in download links, and the ones I generate locally are invalid.

Reply:
How can it be invalid? bazel is using something other than `sha256sum`?